contracts/OrcaProtocol.sol

```diff
@@ -51,12 +51,18 @@ contract OrcaProtocol {
         uint256 _podId,
         uint256 _totalSupply,
         uint256 _votingPeriod,
-        uint256 _minQuorum
+        uint256 _minQuorum,
+        address _gnosisMasterContract
     ) public {
         // add a require to confirm minting was successful otherwise revert
         orcaPodManager.createPod(msg.sender, _podId, _totalSupply);
 
-        orcaVoteManager.createVotingStrategy(_podId, _votingPeriod, _minQuorum);
+        orcaVoteManager.setupPodVotingAndSafe(
```

Comment:
is there a specific reason to tie safe deployment to the vote manager? I feel like it might be worth separating out.

Reply:
Ok - I'll break out. I was thinking that one external call to a contract w/ two internal calls might be cheaper than two external calls, but I don't know if that's actually how it works. I'll try it with them split out and see if there is any major gas impact